astacus/coordinator/plugins/base.py

```diff
@@ -533,7 +536,24 @@ async def clear_delta(
         await cluster.wait_successful_results(start_results=start_results, result_class=ipc.NodeResult)
 
 
-def _prune_manifests(manifests: List[ipc.BackupManifest], retention: Retention) -> List[ipc.BackupManifest]:
+@dataclasses.dataclass
+class ManifestMin:
+    start: datetime.datetime
+    end: datetime.datetime
+    filename: str
+    plugin_data: Dict[str, Any]
```

Comment:
This can be huge for ClickHouse, since it contains the path of all object storage items on remote disk, it shouldn't be part of the `ManifestMin`.